### ISSUE TITLE
No uninstaller provided

### ISSUE THREAD

**Issue by jbattermann** [gmasterscm/tour#2]
Installed gmaster (version today), started it and was greeted with a bare Credentials required screen (did not state why or what the credentials are for & therefore a no-go), wanted to uninstall, noticed that neither in the Windows (10) Control Panel nor in the Start Menu is any entry for Uninstalling gmaster.

How do I get rid of it again, completely.. as in.. what a proper uninstallation would do?

**Comment by psantosl** [comment]
All you need to do is to remove AppData\Local\gmaster. Everything is there. Quite clean :-)

**Comment by NickCraver** [comment]
I was also surprised with the lack of an uninstaller. Removing `AppData\Local\gmaster` is both not intuitive nor complete, for example the start menu entry is still there, GitHub authorizations are left in places, etc. If a product has a complete installer, an uninstaller should accompany. Right clicking on the Windows 10 start menu and clicking "Uninstall" leaves a user lost at Add/Remove programs with a program that's not in the list.

**Comment by evilrix** [comment]
I agree, this is a poor way to convince a user to use your software. I tried gmaster. I don't mind it, but it's not really for me. It's something that I might consider recommending, but not if there's no way to remove it in a user-friendly way.

**Comment by evilrix** [comment]
I agree, this is a poor way to convince a user to use your software. I tried gmaster. I don't mind it, but it's not really for me. It's something that I might consider recommending, but not if there's no way to remove it in a user-friendly way. Plus, manual deletion is hardly clean when it leaves icons behind that must also be located and removed!

**Comment by stamminator** [comment]
I quite like gmaster, but I simply cannot recommend it to colleagues learning git until a proper uninstaller is included.

**Comment by enricoreich** [comment]
After 2 years still no uninstaller. 👎

**Comment by pmanerikar** [comment]
How can I download gmaster? I am lost ..

**Comment by ray-odoo** [comment]
STILL no uninstaller...

**Comment by igroglaz** [comment]
omg..

**Comment by neil-vqa** [comment]
What?!!! still nothing?!

**Comment by JyeGoodlet** [comment]
Seriously?

**Comment by SandersAaronD** [comment]
I don't even know what this program is and I want to fork it to have an uninstaller now.

**Comment by egrdlin** [comment]
2022 still no uninstaller. Is this a joke?

**Comment by stamminator** [comment]
> 2022 still no uninstaller. Is this a joke?

Apparently.
